### PR TITLE
add login page

### PR DESCRIPTION
**Pull request by Trinidad-KianLee**
Good afternoon, Team! I made a html and css file login page

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>MemoirVerse Sign In</title>
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="welcome-section">
+        <h1>Welcome to MemoirVerse!</h1>
+        <p>Enter your personal details to use all of site features.</p>
+      </div>
```
Comment (Escuro-JohnDenielL) [inline frontend/index.html:13]: A line space between the divs would add readability and legibility

Comment (Trinidad-KianLee) [reply frontend/index.html:13]: Alright! I'll be adding lines between my divs for readablity. Thank you!

**Review by Escuro-JohnDenielL — APPROVED** [review]
Looks good!

**Comment by Trinidad-KianLee** [comment]
Thank you for your suggestions! 
Merging pull request uploaded new html and css files and added changes based on review.

**Pull request merged**